pkg/power/acpi/util_linux.go

```diff
@@ -0,0 +1,94 @@
+//go:build linux
+// +build linux
```

Comment:
@rootfs ,those path are linux specific, so we have to split at here for different device?

Reply:
@marceloamaral any idea? maybe a stub here too?